uui-components/src/table/DataTableRow.tsx

```diff
@@ -62,7 +62,7 @@ const DataTableRowImpl = React.forwardRef(function DataTableRow<TItem, TId>(prop
                 rawProps={ {
                     ...params.eventHandlers,
                     role: 'row',
-                    'aria-expanded': props.isFolded == undefined ? undefined : !props.isFolded,
+                    'aria-expanded': props.isFolded === undefined ? undefined : !props.isFolded,
```

Comment:
i guess here we can also have isFolded null value, so let's add this case with null also.